test-mocha/hassio/create_session.spec.ts

```diff
@@ -0,0 +1,47 @@
+import * as assert from "assert";
+import { createHassioSession } from "../../src/data/hassio/supervisor";
+
+const sessionID = "fhdsu73rh3io4h8f3irhjel8ousafehf8f3yh";
+
+describe("Create hassio session", function () {
+  it("Check session without HTTPS", async function () {
+    // @ts-ignore
+    global.document = {};
+    // @ts-ignore
+    global.location = {};
+    await createHassioSession({
+      // @ts-ignore
+      callApi: async function () {
+        return { data: { session: sessionID } };
+      },
+    });
+    assert.deepEqual(
```

Comment:
No need for deepEqual as it is just a string compare?

Reply:
True